WooCommerce/src/main/kotlin/com/woocommerce/android/ui/mystore/MyStoreViewModel.kt

```diff
@@ -231,6 +231,12 @@ class MyStoreViewModel @Inject constructor(
         usageTracksEventEmitter.interacted()
         _selectedRangeType.update { selectionType }
         appPrefsWrapper.setActiveStatsTab(selectionType.name)
+
+        if (isCustomRange) {
```

Comment:
```suggestion
        if (selectionType == SelectionType.CUSTOM) {
```

Minor thing. After the refactor, the code doesn't compile unless I add this.

Reply:
Oops, that was an artefact of a reversed commit. Thanks!